src/Bindings/Plugin.h

```diff
@@ -69,6 +69,7 @@ class cPlugin
 	virtual bool OnKilled                   (cEntity & a_Victim, TakeDamageInfo & a_TDI, AString & a_DeathMessage) = 0;
 	virtual bool OnKilling                  (cEntity & a_Victim, cEntity * a_Killer, TakeDamageInfo & a_TDI) = 0;
 	virtual bool OnLogin                    (cClientHandle & a_Client, UInt32 a_ProtocolVersion, const AString & a_Username) = 0;
+	virtual bool OnLoginForge               (cClientHandle & a_Client) = 0;
```

Comment:
I have a branch that adds a hook for window click and there I had passed a const cItem and got the same problem with boolean. Looked at the code again and have no idea how I fixed that and the changes are small.

Pushing a AStringMap from ManualBindings works fine and a AStringVector in hook works fine, too. I have no idea whats going on there...